homeassistant/components/hydrawise/binary_sensor.py

```diff
@@ -61,14 +58,8 @@ def update(self):
         mydata = self.hass.data[DATA_HYDRAWISE].data
         if self._sensor_type == "status":
             self._state = mydata.status == "All good!"
-        elif self._sensor_type == "rain_sensor":
-            for sensor in mydata.sensors:
-                if sensor["name"] == "Rain":
-                    self._state = sensor["active"] == 1
         elif self._sensor_type == "is_watering":
-            if not mydata.running:
-                self._state = False
-            elif int(mydata.running[0]["relay"]) == self.data["relay"]:
+            if mydata.relays[self.data["relay"] - 1]["timestr"] == "Now":
                 self._state = True
             else:
                 self._state = False
```

Comment:
```suggestion
            self._state = mydata.relays[self.data["relay"] - 1]["timestr"] == "Now"
```

Reply:
Sorry I missed this one on the first pass. Also `relay_data = mydata.relays[self.data["relay"] - 1]` above

Reply:
Thanks. I should have caught that one.